src/csharp/NativeMethods.cs

```diff
@@ -59,17 +59,23 @@ internal class NativeLib
 
         // This function is used to set the input IDs for the generator.
         [DllImport(NativeLib.DllName, CallingConvention = CallingConvention.Winapi)]
-        public static extern unsafe IntPtr /* OgaResult* */ OgaGeneratorParamsSetInputIDs(IntPtr /* OgaGeneratorParams* */ generatorParams,
-                                                                                          int* /* const int32_t* */ inputIDs,
-                                                                                          UIntPtr /* size_t */ inputIDsCount,
-                                                                                          UIntPtr /* size_t */ sequenceLength,
-                                                                                          UIntPtr /* size_t */ batchSize);
+        public static extern IntPtr /* OgaResult* */ OgaGeneratorParamsSetInputIDs(IntPtr /* OgaGeneratorParams* */ generatorParams,
```

Comment:
Same comment as above. We can probably avoid making the change for this function.

Reply:
👍Reverted